deploy-apps/rolling-deploy.html.md.erb

```diff
@@ -90,6 +90,11 @@ The following table describes the limitations to consider when using the beta `v
 		<th>Limitation</th>
 		<th>Description</th>
 	</tr>
+	<tr>
+		<td>Application Manifests</td>
+		<td>The beta `v3-zdt-push` command does not support providing an application manifest via the `-f` flag.  In addition, a `manifest.yml` file in your application directory will be ignored.  The CLI V7 beta will not have this limitation.</td>
```

Comment:
You could also mention that the `v3-apply-manifest` command can be used in the meantime to work around this limitation.